galeracluster/source/gettingstarted.rst

```diff
@@ -3,19 +3,64 @@
 =================
 .. _`getting-started`:
 
-Galera Cluster for MySQL is a synchronous replication solution that can improve availability and performance of MySQL service.  All Galera Cluster nodes are identical and fully representative of the cluster and allow unconstrained transparent ``mysql`` client access, acting as a single-distributed MySQL server.  It provides:
+Galera Cluster is a synchronous replication solution to improve availability and performance of the MySQL database service.  All nodes in Galera Cluster are identical and fully representative of the cluster.  They allow for unconstrained transparent client access, operating as a single, distributed MySQL server.  It provides,
 
 - Transparent client connections, so it's highly compatible with existing applications;
 
 - Synchronous data safety semantics |---| if a client received confirmation, transactions will be committed on every node; and
 
 - Automatic write conflict detection and resolution, so that nodes are always consistent.
 
-Galera Cluster is well suited for :abbr:`LAN (Local Area Network)`, :abbr:`WAN (Wide Area Network)`, and cloud environments.  This Getting Started chapter will help you to get started with a basic Galera Cluster.  You will need root access to three Linux hosts and their IP Addresses.
+Galera Cluster is well suited for LAN, WAN, container and cloud environments.  The following chapters provide you with the basics to setting up and deploying Galera Cluster.  Bear in mind before you get started that you need root access to at least three Linux or FreeBSD hosts and their IP addresses.
+
+.. note:: With the latest release Glaerea Cluster begins the 4.x branch, introducing a number of new releases.  For more information on these features, see :doc:`whatsnew`.
```

Comment:
Typo 'Glaerea'.

Should 'introducing a number of new releases' be 'introducing a number of new features'?